pkg/utils/kube/resource_ops.go

```diff
@@ -267,13 +268,11 @@ func (k *kubectlResourceOperations) newApplyOptions(ioStreams genericclioptions.
 		return nil, err
 	}
 	o.OpenAPISchema = k.openAPISchema
-	o.DryRunVerifier = resource.NewQueryParamVerifier(dynamicClient, k.fact.OpenAPIGetter(), resource.QueryParamFieldValidation)
-	o.FieldValidationVerifier = resource.NewQueryParamVerifier(dynamicClient, k.fact.OpenAPIGetter(), resource.QueryParamFieldValidation)
```

Comment:
Do you know why these fields were removed from the upstream struct? Do we need to be switching to some other mechanism of providing these arguments?

Reply:
I made a similar change in our fork - without much digging, the following seemed to suggest just removing these is the correct approach?

https://github.com/kubernetes/kubernetes/pull/114294/files
https://github.com/kubernetes/kubernetes/pull/114413/files